scripts/amd/gemm/tune_gemm.py

```diff
@@ -571,6 +622,7 @@ def parse_args():
     parser.add_argument("--jobs", type=int, default=1, help="number of generated files")
     parser.add_argument("--iters", type=int, default=1000, help="number of generated files")
     parser.add_argument("--init_type", type=str, default='randn', help="Initialization type for input matrices (default uniform rand [0, 1.0)])")
+    parser.add_argument("--rotating_tensor", type=int, default=256, help="total size of all tensors (default 256MB, need to be larger than the L1, L2, MALL size)")
```

Comment:
so if I don't want to use rotating tensor at all, my choice would be feeding 0 or any value smaller than 1 copy of size A+B+C?

Reply:
Yes, that is right